### ISSUE TITLE
Results input attribute deprecation

### ISSUE THREAD

**Issue by oswaldoferreira** [bower/bower.github.io#262]
Here's the `warning`message on the actual (52.0.2743.116) Chrome version:

```
'results' attribute is deprecated and will be removed in M53, around September 2016. See https://www.chromestatus.com/features/5738199536107520 for more details.
```

And a quick MDN copy/paste on the attribute [1]:

> This is a nonstandard attribute supported by Safari that only applies when the type is search. It is used to control the maximum number of entries that should be displayed in the <input>'s native dropdown list of past search queries. Its value should be a nonnegative decimal integer.

So, I think a quick and wise decision would be to simply remove the attribute.

1: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-results

**Comment by julmot** [comment]
This is the related bug: https://bugs.chromium.org/p/chromium/issues/detail?id=590117

**Comment by julmot** [comment]
Closed via #263.
